pkg/controller/operator/seed/resources/kubermatic/seed-controller-manager.go

```diff
@@ -103,6 +103,14 @@ func SeedControllerManagerDeploymentReconciler(workerName string, versions kuber
 				args = append(args, "-v=2")
 			}
 
+			if cfg.Spec.SeedController.BackupInterval.Duration > 0 {
+				args = append(args, fmt.Sprintf("-backup-interval=%s", cfg.Spec.SeedController.BackupInterval.String()))
```

Comment:
```suggestion
				args = append(args, fmt.Sprintf("-backup-interval=%s", cfg.Spec.SeedController.BackupInterval))
```

see example: https://go.dev/play/p/Y5oRcK5WP5b